x-pack/platform/plugins/shared/streams_app/public/components/data_management/stream_detail_routing/child_stream_list.tsx

```diff
@@ -78,6 +78,17 @@ export function ChildStreamList({ availableStreams }: { availableStreams: string
   const maxNestingLevel = getSegments(definition.stream.name).length >= MAX_NESTING_LEVEL;
   const shouldDisplayCreateButton = definition.privileges.simulate;
   const CreateButtonComponent = aiFeatures && aiFeatures.enabled ? EuiButtonEmpty : EuiButton;
+  const suggestionsRef = useRef<HTMLDivElement>(null);
+
+  useEffect(() => {
+    if (suggestionsRef.current && suggestions) {
+      suggestionsRef.current.scrollIntoView({
```

Comment:
can [this hook](https://github.com/elastic/kibana/blob/bfa3ee530c78fa184a362ae8eb131ae44e7693ec/src/core/packages/chrome/navigation/src/hooks/use_scroll_to_active.ts#L17) be used instead?